usr/tgtd.c

```diff
@@ -613,16 +615,20 @@ static int target_create(const _ha_request *reqp,
 	int rc = 0;
 	char *data = NULL;
 
+	pthread_mutex_lock(&ha_rest_mutex);
```

Comment:
Move the lock call just before the actual tgt command. Do same in other commands too.

Reply:
Done